### PR TITLE
Decrease default value for deregistration_delay

### PR DESCRIPTION
**Pull request by vandrijevik**
As mentioned in https://github.com/counterfind/terraform/pull/51, the deregistration delay configured for an ALB target group has the following meaning:

> Note that you can specify a value of up to 1 hour, and that Elastic Load Balancing waits the full amount of time specified, regardless of whether there are in-flight requests.

(from https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html#deregistration-delay)

To allow maintenance operations to proceed more quickly (so we don’t have to wait a full 5 minutes between taking EC2 instances out of a target group, and shutting down app servers on the instance), this PR updates the default `deregistration_delay` to be 30 seconds (instead of 5 minutes).

### CODE REVIEW

**Review by fedekau — DISMISSED** [review]
I am approving it since is it a pretty simple change. But let's wait for @pturley or @phoolish approval first.

If we merge this, we should also change the SHA for clients using this module and apply it in order to get the benefits.

**Review by pturley — APPROVED** [review]
I agree with Federico. We should update some shas to get this benefit.

**Pull request merged**
